101-cses-multi-role-rdp/metadata.json

```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "https://aka.ms/azure-quickstart-templates-metadata-schema#",
+  "type": "QuickStart",
+  "itemDisplayName": "Blank Template",
+  "description": "Cloud Services (extended support) multi-role deployment with RDP",
+  "summary": "This template allows deploying a Cloud Services (extended support) with mulitple roles. It also creates virtual network, public ip address.",
```

Comment:
This template deploys an Azure Cloud Service (extended support) with multiple roles. It also creates a virtual network and public ip address.